eZ/Publish/API/Repository/Tests/Values/URL/Query/Criterion/SectionTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace eZ\Publish\API\Repository\Tests\Values\URL\Query\Criterion;
+
+use eZ\Publish\API\Repository\Values\URL\Query\Criterion\Section;
+use PHPUnit\Framework\TestCase;
+
+class SectionTest extends TestCase
+{
+    /**
+     * Test a new class and default values on properties.
+     *
+     * @covers \eZ\Publish\API\Repository\Values\Content\Section::__construct
+     */
+    public function testNewClass()
```

Comment:
```suggestion
    public function testInstanceCreation()
```

Reply:
@mikadamczyk
There are 10 other functions named as `testNewClass`, that was the reason I picked it.
May I rename them all or just this one? Or may I leave it as it is?

Reply:
It's not critical to me, especially if we have this name already. It can stay the way it is @ITernovtsiy